cardano-cli/src/Cardano/CLI/EraBased/Options/Governance/Actions.hs

```diff
@@ -99,11 +99,11 @@ pNewCommitteeCmd =
   NewCommitteeCmd
     <$> pNetwork
     <*> pGovActionDeposit
-    <*> pAnyStakeIdentifier
+    <*> pAnyStakeIdentifier Nothing
     <*> pProposalUrl
     <*> pProposalHashSource
-    <*> many pAnyStakeIdentifier
-    <*> many ((,) <$> pAnyStakeIdentifier <*> pEpochNo "Committee member expiry epoch")
+    <*> many (pAnyStakeIdentifier (Just "remove-cc"))
```

Comment:
I think we could use `IsString` here:
```haskell
instance IsString a => IsString (Maybe a) where fromString = Just . fromString
```

this way we could just write `"remove-cc"` instead

Reply:
I'm not sure, generally speaking this instance looks surprising to me.

Also the occurrences of `Just "prefix"` are actually quite rare, most of this PR is just passing the prefix in context around.